packages/suite-storage/.gitignore

```diff
@@ -0,0 +1,97 @@
+# Logs
```

Comment:
same as .eslintrcignore:
do we need to have this file in every package? wouldn't it be better to have it globally?

Reply:
Good catch, I trimmed it down, now it has just 2 lines. 

As I understand it, currently we use one global .gitignore at root level and then one per package which contains only lines specific for this package. This way we can easily tell what is ignored and why. With one global .gitignore containing 200 lines it would be a bit harder to figure out if we can remove some line from it or if some package still needs it.